### DISCUSSION TITLE
Kelleesh24's NTSC-RS Preset Pack | Repository Shortcut

### DISCUSSION THREAD

**Discussion by Creator20Guest** [ntsc-rs/ntsc-rs#200] [Presets]
Oct-4-2024
by Kelleesh24 / Creator20Guest

I forgot completely to upload my presets here on the discussions page...
Because I have uploaded this thing on a new brand GitHub repository...

Just some presets that I made because I was bored and I wanted to make something with this program. The previews to these presets will be shown below with their description. As well as their respective links, to make the things shorter, btw, all the presets are more below

> ![Basic VHS](https://github.com/user-attachments/assets/ea56ab4d-c580-47b1-8342-d78df32e0533)
**Basic VHS**
Just a basic VHS preset
[Basic VHS.json](https://github.com/user-attachments/files/17264196/Basic.VHS.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Broken Clear](https://github.com/user-attachments/assets/7f84f681-90b6-42d5-92e9-22484d66fc36)
**Broken Clear**
A sharper (but imperfect) preset
[Broken Clear.json](https://github.com/user-attachments/files/17264197/Broken.Clear.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Clone](https://github.com/user-attachments/assets/6b0e269c-6024-4d22-9abc-ff86a6186dc5)
**Clone**
Common VHS preset
[Clone.json](https://github.com/user-attachments/files/17264198/Clone.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Clone2](https://github.com/user-attachments/assets/ecc41e41-bb09-47ec-afcf-d99449bce342)
**Clone2**
Uncommon VHS preset
[Clone2.json](https://github.com/user-attachments/files/17264199/Clone2.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Damaged](https://github.com/user-attachments/assets/6e36e2a9-4d87-488f-ad27-3c91aa8c419c)
**Damaged**
Experiment with the ringing function
[Damaged.json](https://github.com/user-attachments/files/17264200/Damaged.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Default](https://github.com/user-attachments/assets/516b838c-91f1-46a0-b916-4d464f74d5ff)
**Default**
ntsc-rs's default preset (why not)
[Default.json](https://github.com/user-attachments/files/17264201/Default.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![General VHS Effect](https://github.com/user-attachments/assets/20d67b8f-cbe7-4dc5-9aa4-09cefab538a4)
**General VHS Effect**
Another common VHS effect
[General VHS Effect.json](https://github.com/user-attachments/files/17264202/General.VHS.Effect.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![HD Television](https://github.com/user-attachments/assets/dfe027d8-bbfd-4c19-bcc2-ddc4325e1e19)
**HD Television**
An HD version of "LQ Television" preset
[HD Television.json](https://github.com/user-attachments/files/17264203/HD.Television.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![i_dont_see_anything](https://github.com/user-attachments/assets/48e91470-49ff-4af1-8f3b-b42e27d6b200)
**i_dont_see_anything**
Literally you can't see anything with this preset
[i_dont_see_anything.json](https://github.com/user-attachments/files/17264204/i_dont_see_anything.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Inverted](https://github.com/user-attachments/assets/6cf67f79-8a8f-4900-89fb-1cd0cb34ece0)
**Inverted**
An experiment with the bandwidth scale function
[Inverted.json](https://github.com/user-attachments/files/17264205/Inverted.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Lost Chroma](https://github.com/user-attachments/assets/3f0e756c-6bda-4b17-9440-9ca4b27518e0)
**Lost Chroma**
An experiment with the chroma loss function
[Lost Chroma.json](https://github.com/user-attachments/files/17264206/Lost.Chroma.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![LQ Television](https://github.com/user-attachments/assets/146ce472-96b0-4e53-a7de-78fc2a275a78)
**LQ Television**
Inspired on jakediditagain style VHS effect (nlo's preset remake)
[LQ Television.json](https://github.com/user-attachments/files/17264207/LQ.Television.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Noise](https://github.com/user-attachments/assets/3194d390-6c2b-4f58-8f61-479fa5c24062)
**Noise**
Experiment with bandwidth scale as well
[Noise.json](https://github.com/user-attachments/files/17264208/Noise.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Old VHS](https://github.com/user-attachments/assets/0b7ea7fb-4247-4eb5-9ef4-cc16a9577fce)
**Old VHS**
Originally made for a song cover, first preset
[Old VHS.json](https://github.com/user-attachments/files/17264209/Old.VHS.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Scanlines](https://github.com/user-attachments/assets/4456d894-702b-4baf-855b-1acf41c9fc6a)
**Scanlines**
Scanline, bandwidth scale function experiment
[Scanlines.json](https://github.com/user-attachments/files/17264210/Scanlines.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![TV Static](https://github.com/user-attachments/assets/f27a8c13-fda2-46b2-aa3f-f637a521b065)
**TV Static**
Just a TV Static
[TV Static.json](https://github.com/user-attachments/files/17264211/TV.Static.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Untitled](https://github.com/user-attachments/assets/e606acda-56b3-4499-bbe5-43f718d351ce)
**Untitled**
Inspired on CapCut, Universe FX, general bad VHS effect
[Untitled.json](https://github.com/user-attachments/files/17264212/Untitled.json)
:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:
![Wavy Image](https://github.com/user-attachments/assets/d7bde5a7-05e0-4ccb-bc1d-9fe9e516090d)
**Wavy Image**
An experiment with the wave function
[Wavy Image.json](https://github.com/user-attachments/files/17264213/Wavy.Image.json)

Here are all the .JSON files... Hope you like these presets, made with love and passion:

> [Basic VHS.json](https://github.com/user-attachments/files/17264196/Basic.VHS.json)
[Broken Clear.json](https://github.com/user-attachments/files/17264197/Broken.Clear.json)
[Clone.json](https://github.com/user-attachments/files/17264198/Clone.json)
[Clone2.json](https://github.com/user-attachments/files/17264199/Clone2.json)
[Damaged.json](https://github.com/user-attachments/files/17264200/Damaged.json)
[Default.json](https://github.com/user-attachments/files/17264201/Default.json)
[General VHS Effect.json](https://github.com/user-attachments/files/17264202/General.VHS.Effect.json)
[HD Television.json](https://github.com/user-attachments/files/17264203/HD.Television.json)
[i_dont_see_anything.json](https://github.com/user-attachments/files/17264204/i_dont_see_anything.json)
[Inverted.json](https://github.com/user-attachments/files/17264205/Inverted.json)
[Lost Chroma.json](https://github.com/user-attachments/files/17264206/Lost.Chroma.json)
[LQ Television.json](https://github.com/user-attachments/files/17264207/LQ.Television.json)
[Noise.json](https://github.com/user-attachments/files/17264208/Noise.json)
[Old VHS.json](https://github.com/user-attachments/files/17264209/Old.VHS.json)
[Scanlines.json](https://github.com/user-attachments/files/17264210/Scanlines.json)
[TV Static.json](https://github.com/user-attachments/files/17264211/TV.Static.json)
[Untitled.json](https://github.com/user-attachments/files/17264212/Untitled.json)
[Wavy Image.json](https://github.com/user-attachments/files/17264213/Wavy.Image.json)

I don't think that I will update this discussion to add new presets. If that occurs, you can visit the GitHub repositories:
**VANILLA:** https://github.com/Creator20Guest/Kelleesh24-s-NTSC-RS-Presets
**EARLY ACCESS:** https://github.com/Kelleesh24/Kelleesh24-s-NTSC-RS-Presets

- Kelleesh24 / Creator20Guest - 2024

**Comment by notdigidani** [comment]
this is awesome

**Reply by Creator20Guest** [reply]
thanks :)

**Comment by Creator20Guest** [comment]
Almost forgot to mention that recently we made a fork where new presets are uploaded
Anyway... Thank you, enjoy...

**Comment by polikux** [comment]
Very large well made preset pack I am impressed :D. Good job! You won't be offended if it is in next version of "NTSC-RT Ultimate Preset Pack?

**Reply by Creator20Guest** [reply]
On the contrary, we will be grateful if they add some of our presets in there. :)
